eng/Version.Details.xml

```diff
@@ -8,9 +8,9 @@
       <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="9.0.0-alpha.1.24161.2">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.24257.1">
```

Comment:
Seems broken to be using 9.0 for `Microsoft.SourceBuild.Intermediate.source-build-externals` but 8.0 for `Microsoft.SourceBuild.Intermediate.source-build-reference-packages`

Reply:
Given this is flowing into 8.0 it seems wrong to be using 9.0 versions of anything.

Reply:
It sounds like the options are:

- Disable the source build leg since 17.10 shouldn't be shipping in source build (as that's the 8.0.3xx SDK). I'm not sure if this would have downstream codeflow impact to other source build legs though.
- Update the 9.0 SBRP to include the 17.3.4 msbuild versions I added and switch back to 9.0 SBRP packages
- Update the externals package to 8.0 and fix any additional SBRP items that come up

Does that some it up?

Reply:
Seems related to https://github.com/dotnet/source-build/issues/3928

Reply:
In that case the change here might be to make sure the externals are 8.0 too.  I wonder if there are darc subscriptions set up incorrectly that are causing 9.0 to flow here.  That said - do we even care about source-build in this branch if we don't ship the SDK versions that consume it?  Should it just be disabled?

Reply:
I pushed an update to the externals package as well to see if that helps. I don't exactly understand all of how SB works so basing this on Eric's comment.

Reply:
Updating to the externals package didn't work. So back to the other options (fix the 8.0 SBRP, switch back to 9.0 and add 17.3.4 there, disable source build)

Reply:
When committed will these changes ever flow into the .NET 9.0 SDK?  If not I think we should just allow the prebuilts.  It is not worth investing anymore effort in this.

```
<UsageData>
  <IgnorePatterns>
    <UsagePattern IdentityGlob="*/*" />
  </IgnorePatterns>
</UsageData>
```

Reply:
@MichaelSimons it looks like this change is needed in Roslyn main as well: https://github.com/dotnet/roslyn/blob/main/src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj#L27

Based on this discussion, it sounds like I need to update SBRP 9.0 version to have 17.3.4 and update here and in roslyn main.

Reply:
I investigated locally.  All prebuilts except one allowed prebuilts as these dependencies exist in the Version.Detail.xml file and non-source-build dependencies.  This means source-build will lift to the latest.  I pushed an update to the prebuilt baseline.  The last prebuilt is System.Threading.Tasks.Dataflow.8.0.0.  This is an indirect prebuilt but I haven't spotted yet where this comes from.  I am still investigating.